sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -16,6 +16,15 @@
 
 namespace Azure.DigitalTwins.Core
 {
+    private const string OTelTwinIdKey = "az.digital_twins.twin.id"
```

Comment:
digital twins were not updated in a while. Should we just remove custom attributes instead?